src/components/client/Client.tsx

```diff
@@ -14,7 +14,7 @@ const Client = ({
 }: ClientPropsInterface): React.ReactElement => {
   const [isExpanded, setIsExpanded] = useState(false);
 
-  const handleClick = (): void => setIsExpanded(!isExpanded);
```

Comment:
shouldn't this remain : void?

Reply:
Adding `:void` here doesn't seem to make the code more robust or readable, so I decided to remove it.